bvm/ballerina-core/src/main/java/org/ballerinalang/model/util/XMLUtils.java

```diff
@@ -655,4 +657,28 @@ private static QName getQName(String localName, String namespaceUri, String pref
         }
         return qname;
     }
+
+    /**
+     * Create an OMElement from an XML fragment given as a string.
+     *
+     * @param xmlFragment the well-formed XML fragment
+     * @return The OMElement created out of the string XML fragment.
+     * @throws XMLStreamException
+     */
+    public static OMElement stringToOM(String xmlFragment) throws XMLStreamException {
+        return stringToOM(OMAbstractFactory.getOMFactory(), xmlFragment);
+    }
+
+    /**
+     * Create an OMElement from an XML fragment given as a string.
+     *
+     * @param omFactory the factory used to build the object model
+     * @param xmlFragment the well-formed XML fragment
+     * @return The OMElement created out of the string XML fragment.
+     * @throws XMLStreamException
+     */
+    public static OMElement stringToOM(OMFactory omFactory, String xmlFragment) throws XMLStreamException {
```

Comment:
This method can be private?

Reply:
BXMLItem.java is using the other method [1]. So IMO we can make this method private

[1]https://github.com/ayomawdb/ballerina/blob/1a19f921ed91531b321937303b0601dc73ff3c23/bvm/ballerina-core/src/main/java/org/ballerinalang/model/util/XMLUtils.java#L668

Reply:
Done